src/DotLiquid.Tests/FilterTests.cs

```diff
@@ -407,12 +407,10 @@ public void TestSort()
             _context["value"] = 3;
             _context["numbers"] = new[] { 2, 1, 4, 3 };
             _context["words"] = new[] { "expected", "as", "alphabetic" };
-            _context["arrays"] = new[] { new[] { "flattened" }, new[] { "are" } };
 
             CollectionAssert.AreEqual(new[] { 1, 2, 3, 4 }, new Variable("numbers | sort").Render(_context) as IEnumerable);
             CollectionAssert.AreEqual(new[] { "alphabetic", "as", "expected" }, new Variable("words | sort").Render(_context) as IEnumerable);
             CollectionAssert.AreEqual(new[] { 3 }, new Variable("value | sort").Render(_context) as IEnumerable);
-            CollectionAssert.AreEqual(new[] { "are", "flattened" }, new Variable("arrays | sort").Render(_context) as IEnumerable);
         }

```

Comment:
Why is sorting nested arrays no longer supported?

Reply:
Checked reference Ruby (same https://replit.com/@microalps/LiquidWhereTests) and it does indeed flatten nested arrays. This is a breaking change.